docs/.vuepress/containers/examples/jsfiddle.js

```diff
@@ -0,0 +1,40 @@
+const JSFIDDLE_ENDPOINT = 'https://jsfiddle.net/api/post/library/pure/';
+
+const { getDependencies } = require('../../handsontable-manager');
+
+const jsfiddle = (id, html, code, css, version, preset) => {
+  const isBabelPanel = preset.includes('react') || preset.includes('vue');
+  const isAngularPanel = preset.includes('angular');
+  const imports = getDependencies(version, preset).reduce(
+    (p, c) =>
```

Comment:
Do I need to go to the `getDependencies()` definition to understand what `p` and `c` are?

Reply:
`p` and `c` are `reduce` callback arguments.

Reply:
It doesn't hold back the initial release. Let's do it at your earliest convenience @Siemienik